doc/history.html

```diff
@@ -134,6 +134,44 @@ <h3>Stable Series</h3>
         <ul>
           <!-- starting 3.6 generated by tags.yaml -->
           <!-- start stable -->
+          <li>
+            <a id="v3_10_14"><b>v3_10_14</b></a> released on June 12, 2025 (<a
```

Comment:
Again the comment about June 12th as release date being different than June 17th in the spec file

Reply:
same